### PR TITLE
Fix signin button add tailwind Class

### PR DESCRIPTION
**Pull request by Dhruvn-patel**
Fixes: #192 

Before Change :
hackError1

After Change:
hackerror1Solve

please Review My PR and replay if any query otherwise thumbsup --> :+1:

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ function Header() {
 
           <div className="flex gap-2 lg:ml-40">
             <button
-              className="btn btn-ghost capitalize"
+              className={`relative  bg-2023-void-2 hover:bg-2023-void-2 btn  text-white rounded-1xl  hover:text-shadow hover:shadow-2023-bavarian-gold-3 `}
```
Comment (max-programming) [inline components/Header.tsx:35]: Can you just switch back to normal string instead of the template literal?

**Review by max-programming — CHANGES_REQUESTED** [review]

File: components/Header.tsx (hunk 1 above)
Comment (max-programming) [reply components/Header.tsx:35]: Also, the border of the button isn't that visible. Can you also fix that?
image

Comment (Dhruvn-patel) [reply components/Header.tsx:35]: @max-programming  Yes I can Do it

Comment (Dhruvn-patel) [reply components/Header.tsx:35]: @max-programming  Can you please Suggest this is okay or not ?
image

Comment (max-programming) [reply components/Header.tsx]: @Dhruvn-patel The previous design was better. Just make the border lighter so that it's visible

**Comment by max-programming** [comment]
Looks good! Thank you Dhruv!

@all-contributors please add @Dhruvn-patel for code, design

**Review by max-programming — APPROVED** [review]

**Pull request merged**